.templates/gpu.sh.template

```diff
@@ -16,20 +16,19 @@
 {% block script %}
     exe ssh abrgpu << EOF
         export PATH="/home/travis-ci/miniconda3/bin:\$PATH"
-        HW_STATUS=0
-        cd /tmp/nengo-dl-$TRAVIS_JOB_NUMBER
-        conda create -q -y -n travis-ci-$TRAVIS_JOB_NUMBER python=$PYTHON_VERSION $TF_VERSION
-        source activate travis-ci-$TRAVIS_JOB_NUMBER
-        pip install $NENGO_VERSION $NUMPY_VERSION
```

Comment:
Why remove `"$NUMPY_VERSION"` here? I know we never actually change it for this script (based on `.travis.yml`), but if someone were to change it there, it should be used here, right?

Reply:
Oh, sorry, I see it moved up above. Got it!